anagrafica/models.py

```diff
@@ -1219,6 +1266,121 @@ def segmenti_collegati(self):
                         attivi.append({'segmento': segmento, 'sede': sede})
         return attivi
 
+    @property
+    def regione(self):
+        sede_riferimento = self.sede_riferimento()
+        if sede_riferimento \
+                and sede_riferimento.estensione == REGIONALE \
+                and sede_riferimento.locazione:
+            return sede_riferimento.locazione.regione
```

Comment:
Starei cauto nell'utilizzare le stringhe parte di `locazione`. Queste vengono dalle Google Maps Geocoding API e sfortunatamente soffrono di una serie di problemi:
- Non abbiamo alcuna garanzia che questo campo sia popolato
- Non abbiamo alcuna garanzia che i valori siano consistenti tra di loro ("Valle D'Aosta", "Aosta", "Valle Di Aosta")
- Non abbiamo alcuna garanzia che l'indirizzo sia corretto (se l'indirizzo non e' presente su Google Maps, abbiamo avuto casi di indirizzi trovati in Africa)

Reply:
Se questo ci serve per ottenere il dominio dell'email, direi di utilizzare un campo opzionale nel modello `Sede` (qualcosa come `dominio_email`). Avrebbe senso navigare ricorsivamente l'albero: se la mia sede di appartenenza locale ha un suo dominio (`catania.sicilia.cri.it`, esempio paradossale), vogliamo usare quello. In caso contrario, vogliamo usare quello del genitore (`sicilia.cri.it`), o ancora `cri.it` per il genitore... Nessun `dominio_email` in tutto l'albero? Nessun dominio email disponibile.

Reply:
ok per questa strada, mi torna. valorizzazione a T0 però manuale? terrei però l'estrazione di questa informazione da dentro la persona... qualcosa tipo

```python
@property
def dominio_email_servizio(self):
    return self.sede_riferimento.primo_dominio_servizio()
```

Reply:
visto il nuovo campo `dominio_email` interessa considerare tutti i valori in questi campi come validi per email di servizio? ovvero vogliamo usare questi domini per capire se un utente ha già un indirizzo di servizio? dall'altro commento di @luca-dex immagino non si possa validare formalmente il valore (non tutti contengono .cri.it)